doc/man5/flux-config-exec.rst

```diff
@@ -38,6 +38,17 @@ sdexec-properties
    (optional) A table of systemd properties to set for all jobs.  All values
    must be strings. See :ref:`sdexec_properties` below.
 
+sdexec-stop-timer-sec
+   (optional) Configure the length of time in seconds after a unit enters
+   deactivating state when it will be sent the sdexec-stop-timer-signal.
+   After the same length of time, if the unit hasn't terminated, for example
+   due to unkillable processes, the unit will be abandoned and the exec RPC
+   is terminated.  Default 30.
```

Comment:
Suggestion: Remind the reader here what triggers the unit entering deactivating state.

Reply:
thanks, fixed that and forced a push.

I forgot that I didn't add tests for the SDEXEC_TIMER stuff.  However, if you are OK with merging this as is I'm fine with that and time permitting, I can follow on with some test improvements in another PR.